keras_cv/layers/serialization_test.py

```diff
@@ -36,54 +36,96 @@ def config_equals(config1, config2):
 
 class SerializationTest(tf.test.TestCase, parameterized.TestCase):
     @parameterized.named_parameters(
-        ("AutoContrast", preprocessing.AutoContrast, {"value_range": (0, 255)}),
-        ("ChannelShuffle", preprocessing.ChannelShuffle, {"seed": 1}),
-        ("CutMix", preprocessing.CutMix, {"seed": 1}),
-        ("Equalization", preprocessing.Equalization, {"value_range": (0, 255)}),
+        ("AutoContrast", preprocessing.AutoContrast, {
+            "value_range": (0, 255)
+        }),
+        ("ChannelShuffle", preprocessing.ChannelShuffle, {
+            "seed": 1
+        }),
+        ("CutMix", preprocessing.CutMix, {
+            "seed": 1
+        }),
+        ("Equalization", preprocessing.Equalization, {
+            "value_range": (0, 255)
+        }),
```

Comment:
Needs fixing?

Reply:
If it's okay I'll fix the formatting errors when all other things are ironed out to avoid redundant efforts.

Reply:
Yes, that's a better approach.

Reply:
Do you know why these errors are occurring yet?

Reply:
Actually, no. I wanted to raise a separate issue regarding this. Not sure if this is due to Windows. In worst case, I'll sync my branch with master and run formatting on colab. I hope that will sort things out.